src/test/java/com/mongodb/jdbc/MongoMock.java

```diff
@@ -62,11 +62,11 @@ protected static void resetMockObjs() throws NoSuchFieldException {
     }
 
     Column generateCol(
-            String database, String table, String column, String bsonType, BsonValue value) {
-        return new Column(database, table, table, column, column, bsonType, value);
+            String database, String table, String column, String bsonType) {
+        return new Column(database, table, table, column, column, bsonType);
     }
 
-    MongoResultDoc generateRow(boolean isEmpty) {
```

Comment:
split this into separate functions for generating the metadata doc and the actual docs representing each row